WordPress/src/main/java/org/wordpress/android/ui/sitecreation/NewSiteCreationMainVM.kt

```diff
@@ -124,10 +151,33 @@ class NewSiteCreationMainVM @Inject constructor(
         }
     }
 
+    fun onPreviewLayoutShown() {
+        isSitePreviewLayoutShown = true
+    }
+
     fun onSitePreviewScreenFinished(createSiteState: CreateSiteState) {
         _wizardFinishedObservable.value = createSiteState
     }
 
+    fun onPositiveDialogButtonClicked(instanceTag: String) {
+        when (instanceTag) {
+            TAG_WARNING_DIALOG -> {
+                tracker.trackFlowExited()
+                _exitFlowObservable.call()
```

Comment:
Should we add a method for `exitFlow` that has the tracker and this call and reuse it in `onBackPressed` as well? It currently looks like the tracker for that one is forgotten which might actually be a deliberate decision. If it's a deliberate decision, we could have a parameter to either track it or not which would convey the intention of the author that it was a deliberate decision not to track the exit in that case. What do you think?

Reply:
Good call! Fixed in #[7f3ab2a](https://github.com/wordpress-mobile/WordPress-Android/pull/9187/commits/7f3ab2aecf60d31fa8c210d63ff34a867c201398)